src/main/resources/assets/nucleus/messages.properties

```diff
@@ -1519,6 +1520,10 @@ command.nucleus.rebuild.start=&aStarting rebuild of the Nucleus user cache.
 command.nucleus.rebuild.fail=&cA rebuild is already in progress.
 command.nucleus.rebuild.end=&aRebuild is complete.
 
+command.near.radiustoobig=&cSupplied radius is over the limit of {0}.
+command.near.playersnear=&ePlayers near {0}.
+command.near.tpto=&cClick to TP to {0}.
```

Comment:
Click -> Click here. TP -> teleport. Also, should be light blue (or at least not red).